src/models/LatentDiffEqModel.jl

```diff
@@ -4,6 +4,13 @@ struct LatentDiffEqModel{M,E,D}
     encoder::E
     decoder::D
 
+    @doc raw"""
+        LatentDiffEqModel(model_type, encoder_layers, decoder_layers)
+    Construct a LatentDiffEqModel.
+    # Arguments
+    model\_type: GOKU() or LatentODE()\
+    encoder\_layers, decoder\_layers: Could be specified manually or produced from default_layers
```

Comment:
I would provide more information about the encoder and decoder layers, such as:

encoder\_layers: contains `feature_extractor`, `pattern_extractor` and `latent_in` layers.
decoder\_layers: contains `latent_out `, `diffeq` and `reconstructor` layers.
To see an example of how to build these layers, please refer to the `default_layers` function.